src/common/cmd.c

```diff
@@ -589,31 +589,31 @@ static cmdstat_t cmd_wait_fn(i32 argc, const char** argv)
 
 static cmdstat_t cmd_cmds_fn(i32 argc, const char** argv)
 {
-    Con_Logf(LogSev_Info, "cmd", "displaying %i commands (`help <cmd>` for more info):", ms_cmds.count);
+    const i32 cmdCount = ms_cmds.count;
+    const char** names = ms_cmds.keys;
+    const cmddef_t* cmds = ms_cmds.values;
 
-    u32* indices = StrDict_Sort(&ms_cmds, SDictStrCmp, NULL);
+    Con_Logf(LogSev_Info, "cmd", "displaying %d commands ('help <cmd>' for more info):", cmdCount);
```

Comment:
If you're displaying all the help text now, then `help <cmd> for more info` seems superfluous. There isn't any additional info displayed from the `help` command, right?

Reply:
Ah, yeah I guess not. It's still nice to have the help command for getting the text for a specific cmd though at least.
I'll amend the text to something like (see also: 'help <cmd>')